res/css/views/elements/_ServerPicker.scss

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 .mx_ServerPicker {
     margin-bottom: 14px;
+    padding-bottom: 16px;
```

Comment:
We have some variables for standardised spacing in `_spacing.scss`, you could use `$spacing-16` here. (margin-bottom is already not playing by the rules but at least the changes can...)
https://www.figma.com/file/X4XTH9iS2KGJ2wFKDqkyed/Compound?node-id=1336%3A18203